biothings/web/handlers/base.py

```diff
@@ -54,7 +54,12 @@ class BaseAPIHandler(BaseHandler, AnalyticsMixin):
     format = 'json'
     cache = 604800  # 7 days
 
-    def initialize(self):
+    def initialize(self,  cache=None):
+        if cache:
+            self.cache = cache
+        if self.cache and isinstance(self.cache, int):
+            # https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control
+            self.set_header("Cache-Control", f"max-age={self.cache}, public")
```

Comment:
You can see in the `RequestHandler` class, the `set_default_headers` is called before the `initialize` method called when so we need to place this block here to guarantee our custom cache works.

Reply:
Make sense. Please add some comments about this, so that we don't have question on this later. Thanks.

Also it's probably worth making a separate `set_cache_header` method to separate out this part. What do you think? @remoteeng00

Reply:
with `set_cache_header` method, it will also allow subclass to override if necessary.

Reply:
it's added